grafana/django-saml2-auth
# CLOPS-574

Purpose

Moved .travis.yml to a Github Actions workflow
Background

Why is this PR needed? Why now? How did it come to our attention?
Migrate the CI workflow to Github Actions
How to Replicate/Test

How can one replicate or test the change to verify that the new code works?
Verify the workflow in ci.yaml file run status
Notes

Any additional relevant info that will help someone do a good review of this PR right now, or 20 months in the future.
References

Links, e.g., to related PRs, relevant sections of software documentation and source code, and stack overflow posts that informed the PR.
https://fool.atlassian.net/browse/CLOPS-574
